pushapkscript/googleplay.py

```diff
@@ -31,9 +33,14 @@ def craft_push_apk_config(context, apks):
     if payload.get('rollout_percentage'):
         push_apk_config['rollout_percentage'] = payload['rollout_percentage']
 
-    # Don't commit anything by default or if dep-signing is detected. Committed
-    # APKs can't be unpublished, unless you push a newer set of APKs.
-    push_apk_config['dry_run'] = True if channel == 'dep' else payload.get('dry_run', True)
+    # Only known channels are allowed to connect to Google Play
+    if not is_allowed_to_push_to_google_play(context):
```

Comment:
tiny nit: I'd send over just the `context.task` since that's the piece used in the function, but it's totally fine this way too, as it's consistent with the rest of the function calls.

Reply:
Yeah, I'm on the fence on this one, too. We can also send `context.task.payload`. I feel like keeping just `context` would help in the future. For instance, if the data moves elsewhere in the context (like in the config), then only the implementation of `is_allowed_to_push_to_google_play()` changes.

Reply:
Makes sense 👍